semgrep/semgrep/evaluation.py

```diff
@@ -137,39 +137,35 @@ def _evaluate_single_expression(
         raise UnknownOperatorError(f"unknown operator {expression.operator}")
 
 
-# Given a `where-python` expression as a string and currently matched metavars,
-# return whether the expression matches as a boolean
 def _where_python_statement_matches(
     where_expression: str, metavars: Dict[str, Any]
 ) -> bool:
     # TODO: filter out obvious dangerous things here
-    output_var = None
+    result = False
 
-    # HACK: we're executing arbitrary Python in the where-python,
-    # be careful my friend
-    vars = {k: v["abstract_content"] for k, v in metavars.items()}
+    local_vars = {k: v["abstract_content"] for k, v in metavars.items()}
```

Comment:
[`vars`](https://docs.python.org/3/library/functions.html#vars) is a builtin Python function, let's avoid using it as a local variable name.